src/core/src/op/lstm_sequence.cpp

```diff
@@ -291,6 +291,14 @@ shared_ptr<Node> op::v0::LSTMSequence::prepare_input(Output<Node> node,
 
 void op::v0::LSTMSequence::validate_and_infer_types() {
     NGRAPH_OP_SCOPE(v0_LSTMSequence_validate_and_infer_types);
+    for (const auto& input : inputs()) {
```

Comment:
@itikhono Do you ok with these changes?
Do we have tests for these changes?

Reply:
looks like the same validation we have for LSTMSequence v5, so it's ok